Xwt/Xwt/ButtonEventArgs.cs

```diff
@@ -80,6 +80,12 @@ public ButtonEventArgs ()
 		/// </summary>
 		/// <value>The number of the presses of the same button.</value>
 		public int MultiplePress { get; set; }
+
+		/// <summary>
+		/// Gets or sets value indicating whether context menu should be shown
+		/// </summary>
+		/// <value><c>true</c> if this context menu should be shown; otherwise, <c>false</c>.</value>
+		public bool ShouldShowContextMenu { get; set; }
```

Comment:
IMHO we should call this property something more like `IsContextMenuTrigger`, because it is describing the nature of the event rather than prescribing what the app should do